src/main/jhipster/src/main/webapp/i18n/en/release-notes.json

```diff
@@ -0,0 +1,12 @@
+{
+    "release-notes": {
+        "title": "Upgrade Notification",
+        "description": "BMS has been upgraded since your last login. See more information below",
```

Comment:
If the user unchecks "dont show again", this description won't be accurate for the next time it sees the notification. Maybe simplify it to just _BMS has been upgraded **recently**. See more information below_

cc @mcrimi @clarysabel

Reply:
Done!

Reply:
We will modify the description as suggested. Agreed with @mcrimi